InstallingDrivers.md

```diff
@@ -70,16 +68,20 @@ Include = /etc/pacman.d/mirrorlist</pre>
 
 Then upgrade the system `sudo pacman -Syu`.
 
-### Nvidia:
+### NVIDIA:
 
-_**Warning**: Please ensure your graphics card is supported by modern Nvidia driver before installing._
+_**Warning**: Please ensure your graphics card is supported by a modern NVIDIA driver before installing._
 _For a list of supported GPUs click here: https://www.nvidia.com/Download/driverResults.aspx/149138/en-us_
 
-Proprietary driver and support for Vulkan are required for proper functionality of games.
+A proprietary driver and support for Vulkan are required for games to function properly. To install the driver package, run:
+
+* `sudo pacman -S nvidia` if you are using the `linux` kernel package.
+* `sudo pacman -S nvidia-lts` if you are using the `linux-lts` kernel package.
+* `sudo pacman -S nvidia-dkms` if you are using a custom kernel package.
 
-To install it, execute following command:
+Then install the other necessary packages by running this command:
 
-    sudo pacman -S nvidia-dkms nvidia-utils lib32-nvidia-utils nvidia-settings vulkan-icd-loader lib32-vulkan-icd-loader
```

Comment:
We want to install nvidia-dkms because the user might be running a custom kernel, such as linux-zen

Reply:
> We want to install nvidia-dkms because the user might be running a custom kernel, such as linux-zen

And what about informing that if you have a custom kernel, you have to install nvidia-dkms, otherwise you don't?

Reply:
I guess separating the command for 2 types of users would be ok.

Reply:
Sorry for the late reply! It's fine by me.